src/main/java/com/google/opensesame/servlets/UserServlet.java

```diff
@@ -55,16 +74,45 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   // Send a user to datastore. Update the current information about the user if one exists.
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userGitHubID = request.getParameter("githubID");
-    if (userGitHubID.trim().isEmpty()) {
-      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Invalid ID");
-      return;
+    String userGitHubAuthToken = request.getParameter("gitHubAuthToken");
+    ArrayList<String> userTags =
+        new ArrayList<>(
+            Arrays.asList(
+                request.getParameterValues("interestTags") == null
+                    ? new String[] {}
+                    : request.getParameterValues("interestTags")));
```

Comment:
nit: Consider breaking this up into multiple lines. At least for me, it's a bit hard to read.

Reply:
will do! I was trying to avoid having to make variables that will only be used once.